src/realworld.md

```diff
@@ -0,0 +1,34 @@
+
+- [CEED participating organizations](https://ceed.exascaleproject.org/about/)
+- [CEED software releases](https://ceed.exascaleproject.org/news/)
+
+## Rodin (Université Grenoble Alpes)
+
+<img src="../img/rodin.png" align="left" style="margin-top: 2px; margin-right: 10px" width="225" alt="colorful simulation on a complex mesh">
+
+[Rodin](https://cbritopacheco.github.io/rodin/) is a lightweight, modular shape, density, and topology optimization framework written in C++, enabling the rapid prototyping of shape optimization algorithms. Rodin provides many of the associated functionalities needed when implementing shape and topology optimization algorithms. These functionalities range from refining and remeshing the underlying shape, to providing elegant mechanisms to specify and solve variational problems. Rodin allows for easy specification and resolution of variational problems, provides uniform class interfaces and transparent conversions between different types of meshes and grid functions, enforces semantic rules related to weak formulations, and adds functionalities of its own on top of wrapped classes. The software library uses MFEM for solving and assembling the linear systems related to the weak formulations.
+
+- [Rodin examples and tutorials](https://cbritopacheco.github.io/rodin/examples-index.html)
+- 2022 MFEM workshop: Carlos Brito Pacheco presents *Rodin: Lightweight and Modern C++17 Shape, Density and Topology Optimization Framework* ([PDF](pdf/workshop22/05_BritoPacheco_Rodin.pdf), [video](https://youtu.be/ZhfDFRJjnU0))
+
+## Wellbore Stability Analysis (OpenSim Technology)
+
+<img src="../img/wellbore.png" align="left" style="margin-top: 2px; margin-right: 10px" width="225" alt="cylindrical wellbore surrounded by a rainbow-colored mesh on gray 3D grid">
+
+[OpenSim Technology](https://opensim.technology/) models subsurface wellbores to prevent collapse. The project’s equations solve for mechanical equilibrium linear elasticity, using PyMFEM to implement the solution. OpenSim’s synthetic well model requires a refined mesh around the wellbore itself to accurately capture stress points, then a coarser mesh farther away from that focal point. The model can compute a second grid of concentric rings, which depict stresses around the cylindrical wellbore and identify failure zones. The team has implemented performance speedups with parallel processing, and they are working on a web app for engineers to use in the field.
```

Comment:
I think this could use some rewording:
models the structural mechanics of subsurface wellbores to prevent collapse. The project uses pyMFEM to solves the equations of linear elasticity for mechanical equalibrium. OpenSim’s synthetic well model requires a refined mesh around the wellbore itself to accurately capture stress points, and uses a coarser mesh farther away from that focal point in order to keep the computational cost down. The model can compute a second grid of concentric rings, which depict stresses around the cylindrical wellbore and identify failure zones. The team continues to develop this code and has recently implemented performance speedups with parallel processing, and they are working on a web app for engineers to use in the field.